rkt/metadata_service.go

```diff
@@ -511,6 +533,7 @@ func handlePodSign(w http.ResponseWriter, r *http.Request) {
 func handlePodVerify(w http.ResponseWriter, r *http.Request) {
 	defer r.Body.Close()
 
+	w.Header().Add("Content-Type", "text/plain; charset=us-ascii")
```

Comment:
got it, thanks. in this case we do have to ensure proper ascii encoding, and reencoding potential utf-8 chars originating from JSON annotations.